posthog/models/filters/mixins/funnel.py

```diff
@@ -71,6 +75,42 @@ def microseconds_from_days(days):
         return microseconds * FunnelWindowDaysMixin.milliseconds_from_days(days)
 
 
+class FunnelWindowMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window(self) -> Optional[int]:
+        _amt = int(self._data.get(FUNNEL_WINDOW, "0"))
+        if _amt == 0:
+            return None
+        return _amt
+
+    @cached_property
+    def funnel_window_interval(self) -> Optional[str]:
```

Comment:
Question about naming: `funnel_window_interval` seems like the window interval, instead of the type?

An alternative:

funnel_window -> funnel_window_interval (how long the window is)
funnel_window_interval -> funnel_window_interval_unit ( in which unit the window is: day/month/minute etc.)